integration_tests/README.md

```diff
@@ -0,0 +1,34 @@
+# Integration Tests
+
+This Cargo package helps us run multi-package tests in our workspace. This setup is inspired by the tokio integration test setup.
+
+## Setup
+
+First setup a local Postgres database and export the postgres database URL as follows:
+```export DATABASE_TEST_URL=postgres://postgres@localhost/<database_name>```
```

Comment:
Nit: would be nice to clarify that the database can be empty and the test framework will initialize the tables and run the migrations.  Just for a new user makes it really clear.